projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -836,6 +836,7 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy {
     public activate(event: FocusEvent) {
         const node = this.selectionNode;
         const shouldEmitSelection = !this.selectionService.isActiveNode(node);
+        this.grid.navigation.activeNode = this.selectionNode;
```

Comment:
Due to some differences in selection node and active node in the multi row layout can omit this assignment or move to to the hierarchical cell activate() method?!